### PR TITLE
266 create tag automatically

### PR DESCRIPTION
**Pull request by mateuszpmroz**
**Info:** 
- It gets tags from the api and translate it

### CODE REVIEW

**Comment by mateuszpmroz** [comment]
[create tag automatically](https://trello.com/c/x6ZhO5J7/266-create-tag-automatically)

File: src/common/localization/en-us.ts [hunk 1]
```diff
@@ -506,5 +506,66 @@ export const english: IAppStrings = {
                 "An error occurred while predicting regions in the current asset. \
                 Please verify your active learning configuration and try again"
         }
+    },
+    wasteTypes: {
```
Comment (mateuszpmroz) [inline src/common/localization/en-us.ts]: @SchnaebeleMarc  In VoTT translations are in typescript file. Do we need to do it in json?

Comment (SchnaebeleMarc) [reply src/common/localization/en-us.ts]: Yes it's better because the tool we use has a large c hoice of file format but not ts files

**Comment by mateuszpmroz** [comment]
@SchnaebeleMarc updated with json files :)

**Pull request merged**